web/profiles/custom/yalesites_profile/modules/custom/ys_core/ys_core.permissions.yml

```diff
@@ -0,0 +1,2 @@
+administer ys_core_footer_settings:
+  title: 'Administer YaleSites Footer Settings'
```

Comment:
Let's remove this file as per https://github.com/yalesites-org/yalesites-project/pull/70/files#r927282639